server/sonar-web/src/main/js/apps/coding-rules/routes.js

```diff
@@ -21,9 +21,9 @@ const routes = [
   {
     indexRoute: {
       getComponent(_, callback) {
-        require.ensure([], require => {
-          callback(null, require('./components/CodingRulesAppContainer').default);
-        });
+        import('./components/CodingRulesAppContainer').then(i =>
+          callback(null, { component: i.default })
```

Comment:
Ok I just saw in the next file that the component should be returned directly in this case.